examples/three-validators/monitoring/prometheus/rules.yml

```diff
@@ -35,4 +35,16 @@
 #          group: 'validators'
 #          repeat_interval: 'long'
 #        annotations:
-#          description: "Participation Rate: {{ $value }} %"
\ No newline at end of file
+#          description: 'Participation Rate: {{ $value }} %'
+#
+#      - alert: LowHealthyValidators
+#        # Condition for alerting
+#        expr: count_validators_alive < 6
```

Comment:
I think the example is with 3 validators and maybe 1 read-only. Also can you please check for different types of alerts? I guess 3 levels of alerting will be fine (one per validator).

Reply:
Should be okay now